docs/development/getting-started.md

```diff
@@ -16,76 +17,77 @@ If you encounter difficulties, please open an issue so that we can make this pro
+Currently, the shoot's nodes run Ubuntu, which does not have the `rsyslog-relp` and `auditd` packages installed, so the configuration done by the extension has no effect.
 Once the shoot is created, we have to manually install the `rsyslog-relp` and `auditd` packages:
 
 ```bash
 kubectl -n shoot--local--local exec -it <name of pod backing the shoot node> -- bash
 ```
 
-Then from inside the node:
+Then, from inside the node, run:
+
 ```bash
 apt-get update && apt-get install -y rsyslog-relp auditd
 systemctl start rsyslog
 ```
 
-Once that is done we can verify that log messages are forwarded to the `rsyslog-relp-echo-server` by checking its logs.
+Once that is done, we can verify that log messages are forwarded to the `rsyslog-relp-echo-server` by checking its logs.
 
 ```bash
 kubectl -n rsyslog-relp-echo-server logs deployment/rsyslog-relp-echo-server
 ```
 
-## Making changes to the Rsyslog Relp Extension
+## Making Changes to the Rsyslog Relp Extension
 
 Changes to the rsyslog relp extension can be applied to the local environment by repeatedly running the `make` recipe.
 
 ```bash
 make extension-up
 ```
 
-## Tearing down the dev environment
+## Tearing Down the Development Environment
 
-To tear down the development environment delete the `Shoot` cluster or disable the `shoot-rsyslog-relp` extension in th `Shoot`'s spec. When the extension is not used by the `Shoot` anymore, you can run:
+To tear down the development environment, delete the shoot cluster or disable the `shoot-rsyslog-relp` extension in the shoot's spec. When the extension is not used by the shoot anymore, you can run:
 
 ```bash
 make extension-down
 ```
 
-This will delete the `ControllerRegistration` and `ControllerDeployment` of the extension, the `shoot-rsyslog-relp-admission` deployment and the `rsyslog-relp-echo-server` deployment.
+This will delete the `ControllerRegistration` and `ControllerDeployment` of the extension, the `shoot-rsyslog-relp-admission` deployment, and the `rsyslog-relp-echo-server` deployment.
 
-# Maintaining the publicly available image for the rsyslog-relp echo server
+## Maintaining the Publicly Available Image for the rsyslog-relp Echo Server
 
 The [testmachinery tests](../../test/testmachinery/shoot/) use an `rsyslog-relp-echo-server` image from a publicly available repository. The one which is currently used is `eu.gcr.io/gardener-project/gardener/extensions/shoot-rsyslog-relp-echo-server:v0.1.0`.
 
 Sometimes it might be necessary to update the image and publish it, e.g. when updating the `alpine` base image version specified in the repository's [Dokerfile](../../Dockerfile#L34).
 
 To do that:
 1. Bump the version with which the image is built in the [Makefile](../../Makefile#L14).
-1. Build the `shoot-rsyslog-relp-echo-server` image:
+2. Build the `shoot-rsyslog-relp-echo-server` image:
```

Comment:
Using just 1. lets you insert bullets without having to change every number after the one you inserted. They will appear as an ordered numbered list when the markdown is rendered. Is there some tooling that requires this change?

Reply:
This is an accidental change due to VS Code formatting and not intended. The numbering does indeed work as you have described. I'll fix it back to the initial layout.